### PR TITLE
revised CryoDy tutorials

### PR DESCRIPTION
**Pull request by SHZ66**

### CODE REVIEW

File: tutorials/cryoem_tutorial/em_analysis.rst [hunk 1]
```diff
@@ -57,70 +57,83 @@ reconstructed model:
...
+      for i in neighbors: 
+         if i not in mapping:
+            mapping.append(i)
+            break
+            
    indices = np.array(mapping)
+   I = np.argsort(indices)
+
+Note that ``indices`` returned from :class:`~sklearn.NearestNeighbors` is a 2-D array with 
+rows corresponding to pseudo-atoms and columns their k-neighbors. After processed by the
```
Comment (jamesmkrieger) [inline tutorials/cryoem_tutorial/em_analysis.rst:91]: Should say "After being processed"

File: tutorials/cryoem_tutorial/em_analysis.rst [hunk 2]
```diff
@@ -57,70 +57,83 @@ reconstructed model:
...
-to the pseudo-atoms based on the mapping:
+Then we create a new :class:`AtomGroup` for the pseudo-atoms based on the mapping, such that 
+they are ordered according to the sequence of residues they are assigned to:
 
 .. ipython:: python
 
-   chid = pdb.getChids()[indices].flatten()
-   emd.setChids(chid)
+   emd2 = AtomMap(emd, I).toAtomGroup()
+   emd2.setResnums(np.arange(emd2.numAtoms()))
```
Comment (jamesmkrieger) [inline tutorials/cryoem_tutorial/em_analysis.rst:110]: How about using the resnums from pmap like you do for chids below?

File: tutorials/cryoem_tutorial/em_analysis.rst [hunk 3]
```diff
@@ -57,70 +57,83 @@ reconstructed model:
...
-   writePDB('EMD-1961_mapped.pdb', amap)
+   writePDB('EMD-1961_mapped.pdb', emd2)
 
 
 Elastic Network Model Analysis
 ==================================================================
 
-Elastic network model analysis can be applied to the pseudo-atomic model as usual:
+Elastic network model analysis can be applied to the pseudo-atomic model as usual, 
+with or without the reordering discussed above:
```
Comment (jamesmkrieger) [inline tutorials/cryoem_tutorial/em_analysis.rst:136]: I can't comment on lines 142 and 144 but I think they should use emd2.

**Review by jamesmkrieger — CHANGES_REQUESTED** [review]
Looks very good now. Just some minor changes

File: tutorials/cryoem_tutorial/em_analysis.rst (hunk 3 above)
Comment (SHZ66) [reply tutorials/cryoem_tutorial/em_analysis.rst:136]: nice catch!

File: tutorials/cryoem_tutorial/em_analysis.rst (hunk 2 above)
Comment (SHZ66) [reply tutorials/cryoem_tutorial/em_analysis.rst:110]: Good idea

**Review by jamesmkrieger — APPROVED** [review]

**Pull request merged**